### DISCUSSION TITLE
How to use with a PyQt application

### DISCUSSION THREAD

**Discussion by noamraph** [faaxm/spix#63] [Q&A]
Hi,

We have a PyQt5 QML application. Could we use Spix to test it?

Thanks a lot, it looks like a great solution!
Noam

**Comment by s-weigand** [comment]
I guess for that to work you would need to make a C++ QML Application, given that all a business logic is implemented inside QML (not sure though).

Did you try [pytest-qt](https://github.com/pytest-dev/pytest-qt)?

**Reply by noamraph** [reply]
The business logic is in Python, in the same process, QML is for the UI.

Perhaps it would be possible to wrap Spix so instead of initializing it from the C++ main function, it would be possible to initialize it from Python.

Thanks,
Noam
